_data/api-swagger.yaml

```diff
@@ -1053,6 +1054,28 @@ definitions:
       For example, in the Netherlands, personal number (BSN) can not be used for privacy reasons
       and is substituted by document number such as passport number, but in other countries this poses no problem.
 
+  OpenID Providers:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      description: |
+        Adds an OpenID verification step to the signing process.
+      properties:
+        ProviderName:
+          type: string
+          description: |
+            (OPTIONAL) When value is provided, it should match the configured one for the organization.
+            If the value does not match then the full list of the organization providers will be given for the end-user to select during the verification step.
+          default: null
+    example: |
+      A OpenID verification response may have the following content:
+      ```json
+      {
+          "Type": "OpenID Providers",
```

Comment:
"Type": "OpenID Providers",

This should/must be `OpenId Connect Providers`

Reply:
This is documentation for an api call that is named OpenID Providers. So changing in documentation will be false and not functional. We named it OpenID Providers.